data/config-schema/src/main/resources/net/corda/schema/configuration/p2p.gateway/1.0/corda.p2p.gateway.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://corda.r3.com/net/corda/schema/configuration/p2p/1.0/corda.p2p.gateway.json",
+  "title": "Corda P2P Gateway Configuration Schema",
+  "description": "Configuration schema for the P2P gateway.",
+  "type": "object",
+  "properties": {
+    "hostAddress": {
+      "description": "The host name or IP address where the HTTP server will bind.",
+      "type": "string",
```

Comment:
Can we use format `hostname` (https://json-schema.org/understanding-json-schema/reference/string.html)